packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/utils.js

```diff
@@ -10,6 +10,28 @@ import { create_derived } from '../../utils.js';
 import is_reference from 'is-reference';
 import { locator } from '../../../../../state.js';
 
+/**
+ * @param {Array<AST.Text | AST.ExpressionTag>} values
+ */
+export function get_states_and_call(values) {
```

Comment:
(and in the corresponding call sites, too)
```suggestion
export function get_states_and_calls(values) {
```

Reply:
Ugh i'm sure at one point i renamed it correctly...fixing lol